### PR TITLE
Fixed https://wso2.org/jira/browse/IOTS-359

### PR DESCRIPTION
**Pull request by menakaj**
1. Integration test pom versions changed.
2. Custom automation extensions for multiple server startup.
Fixes issue [https://wso2.org/jira/browse/IOTS-359](url)

### CODE REVIEW

File: modules/integration/tests-iot-web-ui/src/test/java/org/wso2/carbon/iot/integration/web/ui/test/extension/AnalyticsServerExtension.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    private String executionEnvironment;
+    private AutomationContext automationContext;
+
+
+    @Override
+    public void initiate() throws AutomationFrameworkException {
+        try {
+            automationContext = new AutomationContext("IOT", TestUserMode.SUPER_TENANT_USER);
+            if(getParameters().get(ExtensionConstants.SERVER_STARTUP_PORT_OFFSET_COMMAND) == null) {
+                getParameters().put(ExtensionConstants.SERVER_STARTUP_PORT_OFFSET_COMMAND, "2");
```
Comment (inoshperera) [inline modules/integration/tests-iot-web-ui/src/test/java/org/wso2/carbon/iot/integration/web/ui/test/extension/AnalyticsServerExtension.java:50]: Magic number can be defined

File: modules/integration/tests-iot-web-ui/src/test/java/org/wso2/carbon/iot/integration/web/ui/test/extension/CustomTestServerManager.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+     * This method is called for stopping a Carbon server
+     * <p/>
+     * Add the @AfterSuite annotation in the method overriding this method
+     *
+     * @throws AutomationFrameworkException If an error occurs while shutting down the server
+     */
+    public void stopServer() throws AutomationFrameworkException {
+        carbonServer.serverShutdown(portOffset);
+    }
+
```
Comment (inoshperera) [inline modules/integration/tests-iot-web-ui/src/test/java/org/wso2/carbon/iot/integration/web/ui/test/extension/CustomTestServerManager.java:143]: extra lines

File: modules/integration/tests-iot-web-ui/src/test/java/org/wso2/carbon/iot/integration/web/ui/test/extension/IOTServerExtension.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    private String executionEnvironment;
+    private AutomationContext automationContext;
+
+
+    @Override
+    public void initiate() throws AutomationFrameworkException {
+        try {
+            automationContext = new AutomationContext("IOT", TestUserMode.SUPER_TENANT_USER);
+            if(getParameters().get(ExtensionConstants.SERVER_STARTUP_PORT_OFFSET_COMMAND) == null) {
+                getParameters().put(ExtensionConstants.SERVER_STARTUP_PORT_OFFSET_COMMAND, "0");
```
Comment (inoshperera) [inline modules/integration/tests-iot-web-ui/src/test/java/org/wso2/carbon/iot/integration/web/ui/test/extension/IOTServerExtension.java:50]: is port offset 0 needed? Also need to define the magic number

File: modules/integration/tests-iot-web-ui/src/test/java/org/wso2/carbon/iot/integration/web/ui/test/extension/AnalyticsServerExtension.java (hunk 1 above)
Comment (menakaj) [reply modules/integration/tests-iot-web-ui/src/test/java/org/wso2/carbon/iot/integration/web/ui/test/extension/AnalyticsServerExtension.java:50]: This is fixed in https://github.com/wso2/product-iots/pull/1454/commits/caba24a1e13612fd30c8e2befa5c5d713a64dee8

File: modules/integration/tests-iot-web-ui/src/test/java/org/wso2/carbon/iot/integration/web/ui/test/extension/CustomTestServerManager.java (hunk 2 above)
Comment (menakaj) [reply modules/integration/tests-iot-web-ui/src/test/java/org/wso2/carbon/iot/integration/web/ui/test/extension/CustomTestServerManager.java:143]: This is fixed in https://github.com/wso2/product-iots/pull/1454/commits/caba24a1e13612fd30c8e2befa5c5d713a64dee8

File: modules/integration/tests-iot-web-ui/src/test/java/org/wso2/carbon/iot/integration/web/ui/test/extension/IOTServerExtension.java (hunk 3 above)
Comment (menakaj) [reply modules/integration/tests-iot-web-ui/src/test/java/org/wso2/carbon/iot/integration/web/ui/test/extension/IOTServerExtension.java:50]: This is fixed in https://github.com/wso2/product-iots/pull/1454/commits/caba24a1e13612fd30c8e2befa5c5d713a64dee8

**Pull request merged**